### PR TITLE
RDM-5875 Update ccpay-web-component dependency

### PR DESCRIPTION
**Pull request by danlysiak**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-5875

### Change description ###
Update the ccpay-web-component dependency

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by danlysiak** [comment]
Note latest ccpay-web-component also requires `file-saver` and `xlsx` (confirmed with reporter of RDM-5875).

File: package.json [hunk 1]
```diff
@@ -75,6 +75,7 @@
     "cucumberjs-allure-reporter": "1.0.3",
     "expect": "^23.0.0",
     "express": "^4.16.2",
+    "file-saver": "^2.0.2",
```
Comment (smathangi) [inline package.json]: is it required for pay-web-component?

Comment (danlysiak) [reply package.json]: Yep, the latest version requires both `file-saver` and `xlsx` - fails to compile without.

Comment (hemantt) [reply package.json]: Isnt it an overhead for us to maintain those standalone dependencies versions? @danlysiak @smathangi

Comment (smathangi) [reply package.json]: Agree @hemantt  ideally those should part of pay web-component dependency itself.

File: package.json [hunk 2]
```diff
@@ -58,12 +58,12 @@
     "@angular/router": "~7.0.3",
     "@angular/upgrade": "~7.0.3",
     "@hmcts/ccd-case-ui-toolkit": "2.62.21-RDM-4335-prerelease-2",
-    "@hmcts/ccpay-web-component": "~2.0.50",
+    "@hmcts/ccpay-web-component": "^3.0.0-beta15",
```
Comment (mario-paniccia) [inline package.json]: I've found some references to `ccpay-web-component` in the toolkit as well. A bit old versions thought, so not sure if those are a leftover or still used. Can you please check whether we need to update there as well?

File: package.json [hunk 3]
```diff
@@ -58,12 +58,12 @@
     "@angular/router": "~7.0.3",
     "@angular/upgrade": "~7.0.3",
     "@hmcts/ccd-case-ui-toolkit": "2.62.21-RDM-4335-prerelease-2",
-    "@hmcts/ccpay-web-component": "~2.0.50",
```
Comment (mario-paniccia) [inline package.json]: there's a reference to `ccpay-web-component` also in toolkit. Can you double check whether we have to update also there?  
https://github.com/hmcts/ccd-case-ui-toolkit/blob/c10b9b5bf98754e9f9c938bafc40eaef05ae42f3/package.json#L80

Comment (danlysiak) [reply package.json]: Checked with pay team and @hemantt - shouldn't be required.

File: package.json (hunk 1 above)
Comment (danlysiak) [reply package.json]: 👍 I was advised yesterday that there were issues trying to incorporate into the component itself, but requested they look to sort in future versions so we no longer need to maintain them.

**Pull request closed without merging**